src/app/manage/[clubId]/create/EventCardPreview.tsx

```diff
@@ -0,0 +1,64 @@
+import EventTimeAlert from "@src/components/events/EventTimeAlert";
+import { MoreIcon, PlusIcon } from "@src/icons/Icons";
+import type { RouterOutputs } from "@src/trpc/shared"
+import { format, isSameDay } from 'date-fns';
+import Image from 'next/image';
+
+interface Props {
+	event: RouterOutputs['event']['findByFilters']['events'][number],
+}
+
+const EventCardPreview = ({ event }: Props) => {
+	return (
+		<div className="container flex h-96 w-64 flex-col overflow-hidden rounded-lg bg-white shadow-sm transition-shadow hover:shadow-lg">
+			<div className="relative">
+	  			<div className="h-40 w-96">
+				<Image
+					src={'/event_default.jpg'}
+					alt="event image"
+					fill
+					objectFit="cover"
+				/>
+				<div className="absolute inset-0 p-2">
+					<EventTimeAlert event={event} />
+				</div>
+	  		</div>
+		</div>
+		<div className="flex h-full flex-col p-5">
+	  		<div className="space-y-2.5">
+				<h3 className="font-bold">{event.name}</h3>
+				<h4 className="text-xs font-bold">
+		 		 	<p
+						className="hover:text-blue-primary"
+					>
+						{event.club.name}
+					</p>
+		  <div>
```

Comment:
Add a bit more vertical space between the time and club name to make the card less empty